.github/pull_request_template.md

```diff
@@ -0,0 +1,47 @@
+**Description**
+
+<!-- 
+Please include a summary of the changes and which issue is fixed or which feature it added.
+Please also include relevant motivation and context. List any dependencies that are required for this change.
+-->
+
+<!--
+If Connected to an issue, include:
+Closes # (issue)
+-->
+
+**Type of change**
+
+<!-- Choose those that fit the type of your change and remove the others -->
+
+- Bug fix (change which fixes an issue)
+- New feature (change which adds functionality)
+- Update to functionality
+- Other: <!-- add other type here, e.g. Improve Workflow--> 
```

Comment:
This part should be done via the `Labels`. This would be redundant. We should rather update the possible labels we can assign.

Reply:
ok I see that, I will remove this section